canonical/canonicalwebteam.blog
# Fix blog posts not showing up on u.com/blog

Not entirely sure why, but from [5.4 WordPress API](https://developer.wordpress.org/rest-api/using-the-rest-api/global-parameters/#_embed) doesn't require `POST /posts` to pass `&embed=` with truthy value, it can be an empty string, that will retrieve all embedded media by default. Why passing `true` doesn't? Not sure.

## QA
Code review note: because of the change in API param, all tests have been updated. You don't have to review the 40 other files.

https://ubuntu-com-13396.demos.haus/blog shows now the two missing new blog posts (not in the featured section but below that).
Check https://ubuntu.com/blog for a comparison, the latest 2 posts are missing.
